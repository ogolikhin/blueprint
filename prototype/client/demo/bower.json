{
  "name": "golden layout",
  "version": "0.1.0",
  "dependencies": {
    "jquery": "1.11.1",
    "angular": "1.4.6",
    "angular-bootstrap": "~0.13.4",
    "bootstrap": "~3.3.5",
    "angular-route": "~1.4.6",
    "angular-cookies": "~1.4.6",
    "ag-grid": "~2.1.3",
    "angular-ui-grid": "~3.0.6"
  }
}
