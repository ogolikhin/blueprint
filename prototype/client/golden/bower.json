{
  "name": "golden layout",
  "version": "0.1.0",
  "dependencies": {
    "jquery": "1.11.1",
    "angular": "1.4.6",
    "bootstrap": "~3.3.5",
    "angular-route": "~1.4.6",
    "angular-cookies": "~1.4.6",    
    "golden-layout": "1.0.10",
    "angular-bootstrap": "~0.13.4",
    "bootstrap": "~3.3.5"
  }
}
